package.json

```diff
@@ -49,7 +49,7 @@
     "globby": "11.0.1",
     "istanbul-lib-instrument": "4.0.3",
     "predicates": "2.0.3",
-    "snyk": "1.358.0",
+    "snyk": "1.372.0",
```

Comment:
Please update `tsconfig.json` as well as advised here to get the project building https://github.com/snyk/snyk/issues/1266#issuecomment-671436790